javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/ignore/IgnoredTypesBuilder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.extension.ignore;
+
+/**
+ * This interface defines different ways to ignore/allow instrumenting classes or packages.
+ *
+ * <p>This interface should not be implemented by the javaagent extension developer - the javaagent
```

Comment:
I did not understand this comment. This is extension interface that should not be implemented? What "transformations" is it talking about?

Reply:
🤦  looks like copy-paste mistake -- will fix it.
Thanks!